Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "bevy_ecs_tilemap"
 description = "A tilemap rendering plugin for bevy which is more ECS friendly by having an entity per tile."
-version = "0.4.0"
+version = "0.4.1"
```

Comment:
Let's roll back this change. I push releases manually there is no automated process for that and what's in master is a 0.5 not a 0.4.1 release.

Reply:
I fixed it in my rotate branch.